### ISSUE TITLE
only noise generated

### ISSUE THREAD

**Issue by jiqizaisikao** [ibab/tensorflow-wavenet#322]
I generated the sin wav with pitch 200HZ of 10 S wav file
and trained wavenet with 300 steps ,and the loss down to 0.001,but when i geneated the wave file the out put seems like white  noise!
I tried it sevele times,all the output is noise,why? not enough trainning steps?but the loss is so small ,end the trianning  data is only normal sin wav.

**Comment by devinroth** [comment]
Overtrained. It doesn’t work the way you think it does. You need lots of different data not just one sample. A minor deviation from the original sine way will generate noise. The neural network doesn’t know how to handle the deviation since you only trained it on one scenario. 

Try training with a number of different sine waves of various frequencies and starts.
